dev/com.ibm.ws.jdbc_fat_oracle/publish/servers/com.ibm.ws.jdbc.fat.oracle.ssl/server.xml

```diff
@@ -35,32 +35,36 @@
     <!-- Shared oracle connection properties -->
  	<variable name="oracle.conn.props" value="oracle.net.ssl_version=1.2;oracle.net.ssl_server_dn_match=false;oracle.net.authentication_services=TCPS;" />
         
-    <!-- Datasource using oracle wallet auth and ssl connection -->
-    <variable name="oracle.wallet"            value="oracle.net.wallet_location=${server.config.dir}/security/;" />
-    <variable name="oracle.conn.props.wallet" value="${oracle.conn.props}${oracle.wallet}" /> 
+    <!-- Datasource using oracle wallet SSO auth and ssl connection -->
+    <variable name="oracle.wallet.sso"            value="oracle.net.wallet_location=${server.config.dir}/security/;" />
 
-    <dataSource id="oracleWallet" jndiName="jdbc/oracleWallet">
+    <dataSource id="oracleWalletSSO" jndiName="jdbc/oracleWalletSSO">
     	<jdbcDriver libraryRef="DBLib"/>
-    	<properties.oracle URL="${env.SSL_URL}" connectionProperties="${oracle.conn.props.wallet}"/>
+    	<properties.oracle URL="${env.SSL_URL}" connectionProperties="${oracle.conn.props}${oracle.wallet.sso}"/>
+    </dataSource>
+
+    <!-- Datasource using oracle wallet P12 auth and ssl connection -->
+    <variable name="oracle.wallet.p12"        value="oracle.net.wallet_location=${server.config.dir}/security/;oracle.net.wallet_password=${env.WALLET_PASS};" />
+
```

Comment:
Suggestion for future update.  Could you replace:
```
<variable name="oracle.wallet.p12" value="oracle.net.wallet_location=${server.config.dir}/security/;oracle.net.wallet_password=${env.WALLET_PASS};" />
```
with
```
<variable name="oracle.wallet.p12" value="${oracle.wallet.sso}oracle.net.wallet_password=${env.WALLET_PASS};" />
```
?

Reply:
Yes, we could do that.  But I suspect this test bucket will be used by support to provide examples to customers.  As such, I'd rather be explicit in this case (properties needed for x configuration are....).